postgres/datadog_checks/postgres/postgres.py

```diff
@@ -318,10 +319,17 @@ def dynamic_queries(self):
                 ]
             )
 
-        if self.version < V10:
-            queries.append(QUERY_PG_CONTROL_CHECKPOINT_LT_10)
+        if self.wal_level == 'logical':
+            self.log.debug("wal_level is logical, adding control checkpoint metrics")
+
+            if self.version >= V10:
+                queries.append(QUERY_PG_CONTROL_CHECKPOINT)
+
+            else:
+                queries.append(QUERY_PG_CONTROL_CHECKPOINT_LT_10)
```

Comment:
As mentioned, this is an Aurora specific issue. Calling `pg_current_wal_lsn()` on an Aurora will fail with:
```
SELECT pg_current_wal_lsn();
ERROR:  wal_level must be set to 'logical'
HINT:  WAL control functions cannot be executed when wal_level < logical.
```
Normal PG can run this query regardless of the WAL level so the check should also include `self.is_aurora is False`